### PR TITLE
Update docker-library images

### PR DESCRIPTION
**Pull request by tianon**
- `cassandra`: 2.2.0 (docker-library/cassandra#18)
- `php`: 7.0.0beta2 (docker-library/php#119)
- `rabbitmq`: 3.5.4
- `ruby`: bundler 1.10.6
- `sentry`: default `sentry.conf.py` for easy linking again (docker-library/sentry#15)
- `wordpress`: 4.2.3

### CODE REVIEW

**Comment by tianon** [comment]
Build test of #919; a6174aa3110ac521943fc645b158f51343b2f808 (`cassandra`, `php`, `rabbitmq`, `ruby`, `sentry`, `wordpress`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/cassandra"
$ bashbrew build "$url"
Fetching cassandra (git://github.com/docker-library/cassandra) ...
Processing cassandra:2.0.16 ...
Processing cassandra:2.0 ...
Processing cassandra:2.1.8 ...
Processing cassandra:2.1 ...
Processing cassandra:latest ...
Processing cassandra:2.2.0 ...
Processing cassandra:2.2 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing cassandra:2.0.16
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing cassandra:2.1.8
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing cassandra:2.2.0
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/php"
$ bashbrew build "$url"
Fetching php (git://github.com/docker-library/php) ...
Processing php:5.4.43-cli ...
Processing php:5.4-cli ...
Processing php:5.4.43 ...
Processing php:5.4 ...
Processing php:5.4.43-apache ...
Processing php:5.4-apache ...
Processing php:5.4.43-fpm ...
Processing php:5.4-fpm ...
Processing php:5.5.27-cli ...
Processing php:5.5-cli ...
Processing php:5.5.27 ...
Processing php:5.5 ...
Processing php:5.5.27-apache ...
Processing php:5.5-apache ...
Processing php:5.5.27-fpm ...
Processing php:5.5-fpm ...
Processing php:5.6.11-cli ...
Processing php:5.6-cli ...
Processing php:5-cli ...
Processing php:cli ...
Processing php:5.6.11 ...
Processing php:5.6 ...
Processing php:5 ...
Processing php:latest ...
Processing php:5.6.11-apache ...
Processing php:5.6-apache ...
Processing php:5-apache ...
Processing php:apache ...
Processing php:5.6.11-fpm ...
Processing php:5.6-fpm ...
Processing php:5-fpm ...
Processing php:fpm ...
Processing php:7.0.0beta2-cli ...
Processing php:7.0-cli ...
Processing php:7-cli ...
Processing php:7.0.0beta2 ...
Processing php:7.0 ...
Processing php:7 ...
Processing php:7.0.0beta2-apache ...
Processing php:7.0-apache ...
Processing php:7-apache ...
Processing php:7.0.0beta2-fpm ...
Processing php:7.0-fpm ...
Processing php:7-fpm ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing php:5.4.43-cli
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.4.43-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.4.43-fpm
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'php-ext-install' [4/5]...passed
	'php-fpm-hello-web' [5/5]...passed
testing php:5.5.27-cli
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.5.27-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.5.27-fpm
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'php-ext-install' [4/5]...passed
	'php-fpm-hello-web' [5/5]...passed
testing php:5.6.11-cli
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.6.11-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:5.6.11-fpm
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'php-ext-install' [4/5]...passed
	'php-fpm-hello-web' [5/5]...passed
testing php:7.0.0beta2-cli
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:7.0.0beta2-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'php-ext-install' [4/4]...passed
testing php:7.0.0beta2-fpm
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'php-ext-install' [4/5]...passed
	'php-fpm-hello-web' [5/5]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/rabbitmq"
$ bashbrew build "$url"
Fetching rabbitmq (git://github.com/docker-library/rabbitmq) ...
Processing rabbitmq:3.5.4 ...
Processing rabbitmq:3.5 ...
Processing rabbitmq:3 ...
Processing rabbitmq:latest ...
Processing rabbitmq:3.5.4-management ...
Processing rabbitmq:3.5-management ...
Processing rabbitmq:3-management ...
Processing rabbitmq:management ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing rabbitmq:3.5.4
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing rabbitmq:3.5.4-management
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/ruby"
$ bashbrew build "$url"
Fetching ruby (git://github.com/docker-library/ruby) ...
Processing ruby:2.0.0-p645 ...
Processing ruby:2.0.0 ...
Processing ruby:2.0 ...
Processing ruby:2.0.0-p645-onbuild ...
Processing ruby:2.0.0-onbuild ...
Processing ruby:2.0-onbuild ...
Processing ruby:2.0.0-p645-slim ...
Processing ruby:2.0.0-slim ...
Processing ruby:2.0-slim ...
Processing ruby:2.0.0-p645-wheezy ...
Processing ruby:2.0.0-wheezy ...
Processing ruby:2.0-wheezy ...
Processing ruby:2.1.6 ...
Processing ruby:2.1 ...
Processing ruby:2.1.6-onbuild ...
Processing ruby:2.1-onbuild ...
Processing ruby:2.1.6-slim ...
Processing ruby:2.1-slim ...
Processing ruby:2.1.6-wheezy ...
Processing ruby:2.1-wheezy ...
Processing ruby:2.2.2 ...
Processing ruby:2.2 ...
Processing ruby:2 ...
Processing ruby:latest ...
Processing ruby:2.2.2-onbuild ...
Processing ruby:2.2-onbuild ...
Processing ruby:2-onbuild ...
Processing ruby:onbuild ...
Processing ruby:2.2.2-slim ...
Processing ruby:2.2-slim ...
Processing ruby:2-slim ...
Processing ruby:slim ...
Processing ruby:2.2.2-wheezy ...
Processing ruby:2.2-wheezy ...
Processing ruby:2-wheezy ...
Processing ruby:wheezy ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing ruby:2.0.0-p645
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.0.0-p645-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.0.0-p645-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'ruby-standard-libs' [4/4]...passed
testing ruby:2.0.0-p645-wheezy
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.1.6
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.1.6-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.1.6-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'ruby-standard-libs' [4/4]...passed
testing ruby:2.1.6-wheezy
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.2.2
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.2.2-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing ruby:2.2.2-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'ruby-standard-libs' [4/4]...passed
testing ruby:2.2.2-wheezy
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/sentry"
$ bashbrew build "$url"
Fetching sentry (git://github.com/docker-library/sentry) ...
Processing sentry:7.6.2 ...
Processing sentry:7.6 ...
Processing sentry:7 ...
Processing sentry:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing sentry:7.6.2
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/a6174aa3110ac521943fc645b158f51343b2f808/library/wordpress"
$ bashbrew build "$url"
Cloning wordpress (git://github.com/docker-library/wordpress) ...
Processing wordpress:4.2.3-apache ...
Processing wordpress:4.2.3 ...
Processing wordpress:4.2-apache ...
Processing wordpress:4.2 ...
Processing wordpress:4-apache ...
Processing wordpress:apache ...
Processing wordpress:4 ...
Processing wordpress:latest ...
Processing wordpress:4.2.3-fpm ...
Processing wordpress:4.2-fpm ...
Processing wordpress:4-fpm ...
Processing wordpress:fpm ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing wordpress:4.2.3-apache
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing wordpress:4.2.3-fpm
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

File: library/cassandra [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-2.0: git://github.com/docker-library/cassandra@711f62f57e181eb373910246e27036aaa14eb54f 2.0
+2.0.16: git://github.com/docker-library/cassandra@b9b173c523af2da70fdd898565a556af12d6b23a 2.0
+2.0: git://github.com/docker-library/cassandra@b9b173c523af2da70fdd898565a556af12d6b23a 2.0
 
-2.1.8: git://github.com/docker-library/cassandra@3a63378f4be40687816258e1cb9f96e2dd76104f 2.1
-2.1: git://github.com/docker-library/cassandra@3a63378f4be40687816258e1cb9f96e2dd76104f 2.1
-latest: git://github.com/docker-library/cassandra@3a63378f4be40687816258e1cb9f96e2dd76104f 2.1
+2.1.8: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
+2.1: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
+latest: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
```
Comment (tianon) [inline library/cassandra]: crap, shouldn't this be on `2.2` now?

Comment (yosifkit) [inline library/cassandra]: oh, probably yes... :unamused:

Comment (tianon) [inline library/cassandra]: https://github.com/docker-library/cassandra/commit/e21f9c5557a82107c70b2d3b29caa10c15248782

```patch
diff --git a/library/cassandra b/library/cassandra
index 7bd113c..10b4f60 100644
--- a/library/cassandra
+++ b/library/cassandra
@@ -5,7 +5,8 @@
 
 2.1.8: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
 2.1: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
-latest: git://github.com/docker-library/cassandra@1a2fa7d60b9fca1135df05e3b0f7795d3485e2d6 2.1
 
 2.2.0: git://github.com/docker-library/cassandra@081e3c47f144279ac5fe16a024581f3fd9b62521 2.2
 2.2: git://github.com/docker-library/cassandra@081e3c47f144279ac5fe16a024581f3fd9b62521 2.2
+2: git://github.com/docker-library/cassandra@081e3c47f144279ac5fe16a024581f3fd9b62521 2.2
+latest: git://github.com/docker-library/cassandra@081e3c47f144279ac5fe16a024581f3fd9b62521 2.2
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**